docs/source/_static/vendor/styles.css

```diff
@@ -102,12 +102,12 @@ em {
     cursor: pointer;
 }
 
-#navbar-item {
```

Comment:
removed the duplicate id from navbar item and converted the CSS to follow the class